setup/so-setup

```diff
@@ -335,7 +340,8 @@ if ! [[ -f $install_opt_file ]]; then
 			"MNIC=$MNIC" \
 			"HOSTNAME=$HOSTNAME" \
 			"MSRV=$MSRV" \
-			"MSRVIP=$MSRVIP" > "$install_opt_file"
+			"MSRVIP=$MSRVIP" \
+			"NODE_DESCRIPTION=$NODE_DESCRIPTION" > "$install_opt_file"
```

Comment:
Does this NODE_DESCRIPTION eventually make it into the sensoroni:node_description pillar?

This is how it gets used, in sensoroni.json:

```
{%- set DESCRIPTION = salt['pillar.get']('sensoroni:node_description') %}
```

Reply:
Yes it does, see [line 1431 in dev](https://github.com/Security-Onion-Solutions/securityonion/blob/c60d4aca16e8ca0ae3195533085eb3f31debbf25/setup/so-functions#L1431)